doc/source/examples/integrated_gradients_transformers.ipynb

```diff
@@ -43,18 +43,22 @@
     "import numpy as np\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # choose whether to use the BERT pr distilBERT model by selecting the appropriate name
</pre><p>pr -&gt; or</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/717/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>